### ISSUE TITLE
this.popover.addEventListener is not a function

### ISSUE THREAD

**Issue by ansarizafar** [flaming-cl/editorjs-text-color-plugin#21]
I am trying to add color plugin to editorjs config but when I select a text block I get this error.

![image](https://github.com/flaming-cl/editorjs-text-color-plugin/assets/5788284/4c482168-da07-41c7-93f6-de9c52b87a0a)

```javascript
  Color: {
      class: ColorPlugin, // if load from CDN, please try: window.ColorPlugin
      config: {
         colorCollections: ['#EC7878','#9C27B0','#673AB7','#3F51B5','#0070FF','#03A9F4','#00BCD4','#4CAF50','#8BC34A','#CDDC39', '#FFF'],
         defaultColor: '#FF1300',
         type: 'text', 
         customPicker: true // add a button to allow selecting any colour  
      }     
    },
```

**Comment by dogyun-wildseed** [comment]
I have same problem

**Comment by fmabid** [comment]
So, the problem is with the new version of Chrome 114. I tested in FireFox and Chrome-113, there was no issue. 
@flaming-cl if possible then please look into this. Would be grateful. Thanks for this nice plugin.

**Comment by flaming-cl** [comment]
@dogyun-wildseed @ansarizafar @fmabid Hi, do you mind to provide more details about this issue? 
Here is a template for adding details. This will help us to better understand this issue, thanks.

### Editorjs Version
2.25

### Other unofficial Editorjs plugins you are using
If not, leave this blank

### Device
PC

### Operating System
Ubuntu / Windows / Mac

### Browser
Chrome 114?

**Comment by dogyun-wildseed** [comment]
Yes, I have tested with most recent version. Not sure I could reproduce with in old eidtorjs version. (I will bring more info later)

## Editorjs Version
2.27

## Other unofficial Editorjs plugins you are using
- editorjs-undo
- I have a few of my custom tools for Editorjs. But none of them are inline tools

## Device
M1 Mac book pro

## Operating System
Mac

## Browser
Chrome  Version 114.0.5735.106 (Official Build) (arm64)

**Comment by dogyun-wildseed** [comment]
I can confirm it happend with `Editorjs: 2.25` as well

**Comment by fmabid** [comment]
### Editorjs Version
2.27.0

### Other unofficial Editorjs plugins you are using

- editorjs-text-alignment-blocktune
- editorjs-inline-font-size-tool

### Device
Lenovo IdeaPad S540

### Operating System
Ubuntu

### Browser
Version 114.0.5735.90 (Official Build) (64-bit)

**Comment by peeter2** [comment]
I have the same problem. Please fix!

**Comment by iwnow** [comment]
Just need rename in src\picker\index.js
this.popover to this.$popover
Seems conflict with chrome popover implementation

**Comment by flaming-cl** [comment]
Fixed in v2.0.3. Sorry for the inconvenience caused by this.
